pokesets/Anime/Ash.yaml

```diff
@@ -530,6 +568,25 @@ evs: {hp: 0, atk: 144, def: 0, spA: 252, spD: 0, spe: 112}
 ivs: 31
 tags: [ash, anime]
 ---
+species: Lucario
+setname: Ash
+ability: [Inner Focus]
+item: [null]
+ball: [Poke]
+moves:
+- [Aura Sphere]
+- [Force Palm]
+- [Double Team]
+- [Metal Claw]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Adamant
+evs: {hp: 0, atk: 252, def: 0, spA: 144, spD: 0, spe: 112}
+ivs: 31
+tags: [ash, anime]
+  ---
```

Comment:
this file contains a few of these. The `---` must not be indented

Reply:
Fixed.